zerver/tests/test_signup.py

```diff
@@ -1510,6 +1510,25 @@ def test_accessing_invites_in_another_realm(self) -> None:
         error_result = self.client_delete('/json/invites/' + str(prereg_user.id))
         self.assert_json_error(error_result, "No such invitation")
 
+    def test_check_user_able_to_register_completely(self) -> None:
+        user = self.nonreg_email('alice')
+        password = "password"
+
+        confirmation_link = create_confirmation_link(user, 'host', Confirmation.USER_REGISTRATION)
```

Comment:
The idea in this test seems about right. But here, ``create_confirmation_link`` can't be called on an email. You want to give a PreregistrationUser object here. https://github.com/zulip/zulip/pull/13462/files#diff-6320ba8b34ca87c2f50bae6f29e1c08dR1060 is an example of how you can use ``do_invite_users`` to get a new PreregUser generated and then get a reference to it.

Reply:
Sure! I will look into it.